index.md

```diff
@@ -4,23 +4,41 @@ layout: default
 
 _“Best practice templates and writing instructions for documenting open source software.”_
 
+### Why?
 
-### Download
+We want to help open source projects be more effective and more impactful by helping them lift the quality of their documentation. In the process, we expect to lift the quality of technical writing for all software.
```

Comment:
what's the difference between effective and impactful here? :) Probably better to just leave as "more effective". 

Also I think better to take out the "In the process ..." sentence. We may expect this but it's kind of a lofty goal. Let's underpromise and overdeliver! :)

Reply:
Hmm, Cameron added that sentence in. He has lofty goals :-)  I'll remove it and let you two duke it out :-)